html/e79e55c8-5f3a-4cd4-a2b3-dc67a99afa5f.html

```diff
@@ -93,7 +93,7 @@ <h4>Lesson Vocabulary</h4>
   </tbody>
 </table>
 <br>
-<div class="os-raise-graybox"><h4>Support for English Language Learners</h4>
+=<h4>Support for English Language Learners</h4>
```

Comment:
@susanescobar66 - It looks like the validator is failing because of this stray `=` character. Removing that should fix things.